website/project/model.py

```diff
@@ -4241,3 +4250,33 @@ def validate_metadata(self, *args, **kwargs):
         Validates draft's metadata
         """
         return self.registration_schema.validate_metadata(*args, **kwargs)
+
+
+class PreprintProvider(StoredObject):
```

Comment:
To avoid further clutter to this file, consider moving model to `website/preprints/`. Not a blocker